blueprints/app/files/.eslintrc.js

```diff
@@ -33,13 +33,21 @@ module.exports = {
       env: {
         browser: false,
         node: true,
-      }
+      },<% if (isAddon) { %>
+      plugins: ['node'],
+      rules: Object.assign(
```

Comment:
can we not use `extends` here?

Reply:
Sadly no. You cannot use extends inside of overrides yet. 

This is being tracked in https://github.com/eslint/eslint/issues/8813.